modules/flake-parts/packages.default.nix

```diff
@@ -13,8 +15,34 @@
       reverse-proxy-nix-cache = pkgs.writeShellScriptBin "reverse-proxy-nix-cache" ''
         sudo ${pkgs.caddy}/bin/caddy reverse-proxy --from :80 --to :5000
       '';
+
+      turn-readiness-check = pkgs.writeShellApplication {
+        name = "turn-readiness-check";
+        runtimeInputs = [
+          self'.packages.tx5
+        ];
+        text =
+          ''
+            set -e
+          ''
+          + builtins.concatStringsSep "\n" (builtins.map (
+              name: ''
+                echo "### checking ${name}... ###"
+                set -x
+                turn-stress ${name}.infra.holochain.org 443 test test
+                turn_doctor wss://${self.nixosConfigurations.${name}.config.services.tx5-signal-server.url}
```

Comment:
@neonphog are these appropriate or shall i use the new tool introduced in https://github.com/holochain/holochain/pull/3496 ? 

the goal here is not even a full-blown health-check, just a readiness check. i don't have a well-defined line between readiness and health but i'm interested in establishing one.

Reply:
I guess turn_doctor implicitly validates the signal server. You might at least want to use the new tool to check the bootstrap. Maybe it'd be clearer what was going if you just use the new tool across the board, but doesn't make a huge difference.

Reply:
i'll use that once it's merged :+1:

Reply:
merged :+1:

Reply:
created a follow-up issue to track the switch